deploy/helm/sumologic/upgrade-1.0.0.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+
+MAN="Thank you for upgrading to v1.0.0 of the Sumo Logic Kubernetes Collection Helm chart.
+As part of this major release, the format of the values.yaml file has changed.
+
+This script will automatically take the configurations of your existing values.yaml
+and return one that is compatible with v1.0.0.
+
+Usage:
+  ./upgrade-1.0.0.sh /path/to/values.yaml
+
+Returns:
+  new_values.yaml
+
+For more details, please refer to Migration steps and Changelog here: [link]
+"
+
+if [ "$1" = "" ]
+then
+  echo "$MAN"
+  exit 1
+fi
+
+OLD_VALUES_YAML=$1
+
+URL=https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/4bd5a737b0f9730e64c5d59f168420af73e81a9b/deploy/helm/sumologic/values.yaml
+curl -s $URL > new.yaml
```

Comment:
Maybe we could use link to release branch here?

Reply:
For sure, I left a TODO review comment to myself above but can update that now